### PR TITLE
Use style repo urls for knode site

### PR DESCRIPTION
**Pull request by hackygolucky**
This change uses knode/style urls as links. It also updates some awkward copy.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,16 +18,247 @@
...
           <h1 id="project_title">KNODE</h1>
-          <h2 id="project_tagline">A portal to provide a comfy living room for Node.js community members looking for/to provide resources, speakers, and guidance. </h2>
+          <h2 id="project_tagline">A portal to provide a comfy living room for
+            Node.js community members looking for/to provide resources, speakers, and
+            guidance. </h2>
 
         </header>
     </div>
-
+    -->
```
Comment (chrisdickinson) [inline index.html]: this is all commented out?

File: index.html [hunk 2]
```diff
@@ -7,6 +7,10 @@
     <meta name="description" content="KNODE : A portal to provide a comfy living room for Node.js community members looking for/to provide resources, speakers, and guidance. " />
 
     <link rel="stylesheet" type="text/css" media="screen" href="stylesheets/stylesheet.css">
```
Comment (chrisdickinson) [inline index.html:9]: should we blow this away?

File: index.html [hunk 3]
```diff
@@ -7,6 +7,10 @@
...
+    <link rel="stylesheet" href="http://knode.io/style/assets/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="http://knode.io/style/assets/css/ostrich-sans/stylesheet.css" />
+    <link rel="stylesheet" href="http://knode.io/style/assets/css/main.css" />
+    <link rel="stylesheet" type="text/css" media="screen" href="stylesheets/site-specific.css">
```
Comment (chrisdickinson) [inline index.html]: `type="text/css"` should be unnecessary in this position.

File: index.html (hunk 2 above)
Comment (hackygolucky) [inline index.html:9]: tried it before and it went wonky. Was erring on the side of caution until otherwise okay.

File: index.html [hunk 4]
```diff
@@ -14,16 +18,247 @@
...
+                    L128,34
+                    Q128 32,124 30
+                    L68,2
+                    Q64 0,60 2
+                    L4,30
+                    Q0 32,0 34
+                    L0,32
+                    Z
+                " />
+                -->
```
Comment (wraithan) [inline index.html:92]: @chrisdickinson commented out paths in the SVG... wat?

File: index.html (hunk 1 above)
Comment (wraithan) [inline index.html]: Looks like L250 to L261: https://github.com/knode/knode.github.io/pull/3/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR250

File: index.html (hunk 4 above)
Comment (hackygolucky) [inline index.html:92]: can we file this as an issue separately of this PR?

Comment (wraithan) [inline index.html:92]: I'm fine with that.

**Comment by wraithan** [comment]
:+1: from me.

File: index.html (hunk 4 above)
Comment (chrisdickinson) [inline index.html:92]: @wraithan remember, SVG was handwritten :)

**Comment by chrisdickinson** [comment]
:+1:

**Pull request merged**